phylorthology/samplesheet_test.csv

```diff
@@ -0,0 +1,7 @@
+species,file,taxonomy,shallow,broad,mode,uniprot,mcl_test
+Entamoeba_histolytica,https://github.com/Arcadia-Science/test-datasets/raw/main/phylorthology/Entamoeba_histolytica-test-proteome.fasta,Amoebozoa,eukaryota_odb10,eukaryota_odb10,proteins,true,false
```

Comment:
I'm a bit confused - why do we have the test data (fasta files) in the phylorthology repository if we're just specifying that we should pull the test data from the arcadia-science test datasets repository?